pkgs/by-name/ht/httping/package.nix

```diff
@@ -0,0 +1,71 @@
+{
+  cmake,
+  fetchFromGitHub,
+  fftw ? null,
```

Comment:
```suggestion
  fftw,
```
that's not null

Reply:
Thanks for the review! I removed this.

I didn't delete it when reformatting, because I don't understand why this code was there in the first place.